### ISSUE TITLE
Removed underscore even if wrapped in code markup.

### ISSUE THREAD

**Issue by tateisu** [huntertran/markdown-toc#8]
I wrote heading like as:
```
#### `aaa_decode_addbytes()`
```

auto-markdown-toc generates TOC line like as:
```
- [`aaadecodeaddbytes()`](#aaadecodeaddbytes)
```

What's wrong:
- underscore is gone.
- link is not correct.

Expected behavior
- underscore NOT gone.
- correct link.

**Comment by tateisu** [comment]
I'm using https://github.com/showdownjs/showdown to generate HTML.

**Comment by huntertran** [comment]
Interesting. I will look into this issue this weekend.

**Comment by maxbublik** [comment]
I happened to have the same issue. Underscores are gone, linkes are broken.
I usually document database tables, TOCs are crucial since files are long and underscores are quite often used in table names.

**Comment by huntertran** [comment]
I believe this issue is fixed in version 3.0.10
